bundler/lib/bundler/version.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: false
 
 module Bundler
-  VERSION = "2.7.0.dev".freeze
+  VERSION = (ENV["BUNDLER_3_MODE"] == "true" ? "3.0.0" : "2.7.0.dev").freeze
```

Comment:
Not sure how beneficial that could be, but what about to make it possible to override version at all with like `BUNDLER_OVERRIDE_VERSION=3.1.0`?

Reply:
All good!